src/common/country-base.js

```diff
@@ -27,7 +27,6 @@ const eu_countries = [
     'se',
     'ie',
     'im',
-    'gb',
```

Comment:
We can't remove this from here as most of the regulatory contents follow EU/UK views together and by removing this, a client will not see the EU view if they are coming from GB.